app/javascript/src/LoginPage/RequestPasswordWrapper.jsx

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import React from 'react'
+import { createReactWrapper } from 'utilities/createReactWrapper'
+
+import { LoginPage } from '@patternfly/react-core'
+import { RequestPasswordForm, FlashMessages } from 'LoginPage'
+
+import 'LoginPage/assets/styles/loginPage.scss'
+
+import brandImg from 'LoginPage/assets/images/3scale_Logo_Reverse.png'
+import PF4DownstreamBG from 'LoginPage/assets/images/PF4DownstreamBG.svg'
+
+import type { FlashMessage } from 'Types'
+
+type Props = {
+  flashMessages: Array<FlashMessage>,
+  providerLoginPath: string,
+  providerPasswordPath: string
+}
+
+const RequestPassword = ({ flashMessages, providerLoginPath, providerPasswordPath }: Props) => (
+  <LoginPage
+    brandImgSrc={brandImg}
+    brandImgAlt="Red Hat 3scale API Management"
+    backgroundImgSrc={PF4DownstreamBG}
+    backgroundImgAlt="Red Hat 3scale API Management"
+    loginTitle={`Request Password`}
+    footer={null}
```

Comment:
Is it possible not to pass this prop? (is it default to null?)

Reply:
You are right, default is null